source/read/change-streams.txt

```diff
@@ -0,0 +1,284 @@
+  specified document
+- ``$redact``: Restricts the contents of the documents
+- ``$unset``: Removes fields from documents
+
+The {+driver-short+} provides the ``Aggregates`` class, which includes helper methods
+for building the preceding pipeline stages. 
+
+.. tip::
+   
+   To learn more about pipeline stages and their corresponding ``Aggregates``
+   helper methods, see the following resources:
+   
+   - :manual:`Aggregation Stages </reference/operator/aggregation-pipeline/>` in the
+     {+mdb-server+} manual
+   - `Aggregates <{+api+}/org/mongodb/scala/model/Aggregates$.html>`__ in the API
+     documentation
+
+The following example creates a pipeline that uses the ``Aggregates.filter()`` method
+to build the ``$match`` stage. Then, the code passes this pipeline to the ``watch()``
+method and instructs ``watch()`` to output events only when update operations occur:
+
+.. literalinclude:: /includes/read/change-streams.scala
+   :start-after: start-change-stream-pipeline
+   :end-before: end-change-stream-pipeline
+   :language: scala
+   :dedent:
+
+Modify watch() Behavior
+-----------------------
+
+You can modify the behavior of the ``watch()`` method by chaining
+methods provided by the ``ChangeStreamObservable`` class. The following
+table describes some of these methods:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``fullDocument()``
+     - | Specifies whether to show the full document after the change, rather
+         than showing only the changes made to the document. To learn more about
+         this option, see the :ref:`scala-change-stream-pre-post-image` section of this
+         guide.
+
+   * - ``fullDocumentBeforeChange()``
+     - | Specifies whether to show the full document as it was before the change, rather
+         than showing only the changes made to the document. To learn more about
+         this option, see :ref:`scala-change-stream-pre-post-image`.
+
+   * - ``comment()``
+     - | Attaches a comment to the operation.
+
+   * - ``startAtOperationTime()``
+     - | Instructs the change stream to provide only changes that occurred at or after
+         the specified timestamp.
+
+   * - ``collation()``
+     - | Sets the collation to use for the change stream cursor.
+
+For a full list of ``watch()`` options, see `ChangeStreamObservable
+<{+api+}/org/mongodb/scala/ChangeStreamObservable.html>`__ in the API
+documentation.
+
+.. _scala-change-stream-pre-post-image:
+
+Include Pre-Images and Post-Images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+   You can enable pre-images and post-images on collections only if your
+   deployment uses {+mdb-server+} v6.0 or later.
+
+By default, when you perform an operation on a collection, the
+corresponding change event includes only the modified fields and
+their values before and after the operation. 
+
+You can instruct the ``watch()`` method to return the document's **pre-image**, or the
```

Comment:
Q: is this returned in addition to the modified fields, or instead of them? probably worth saying. 

also applies to post-image